CONTRIBUTING.md

```diff
@@ -1,18 +1,26 @@
 # Contributing to Dogecoin Core
 
-The Dogecoin Core project operates an open contributor model where anyone is
-welcome to contribute towards development in the form of peer review, testing
-and patches. This document explains the practical process and guidelines for
-contributing.
+Dogecoin Core is an open source project, and we would welcome contributions which provably
+improve the state of the software. As a pointer on where to start, please look at the
+[projects](https://github.com/dogecoin/dogecoin/projects) currently underway,
+where we track issues which are being worked on or need attention from others.
 
-Firstly in terms of structure, there is no particular concept of "Core
-developers" in the sense of privileged people. Open source often naturally
-revolves around meritocracy where longer term contributors gain more trust from
-the developer community. However, some hierarchy is necessary for practical
-purposes. As such there are repository "maintainers" who are responsible for
-merging pull requests as well as a "lead maintainer" who is responsible for the
-release cycle, overall merging, moderation and appointment of maintainers.
+## Branch Strategy
```

Comment:
I would speak later about Branch strategy, more general information should come first, like `Community`.

We probably have to manage different type of user, it is relevant only to devs, so it's possible not the best place.
[Think like people should mainly get useful information].

Reply:
So the idea was to focus on topics we see most issues with, but yes can move it down a section.

Reply:
Done